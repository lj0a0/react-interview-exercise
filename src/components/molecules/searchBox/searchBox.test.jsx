import React from "react";
import SearchBox from "./searchBox";
import renderer from "react-test-renderer";


test("SearchBox snapshot", () => {
  const component = renderer.create(
    <SearchBox
        errorCode={{code: 0}}
        githubSearch={"react"}
        githubSearchOption={"star"}
    />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});