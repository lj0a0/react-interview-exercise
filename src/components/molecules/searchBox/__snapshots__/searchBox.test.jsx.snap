// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBox snapshot 1`] = `
<section
  className="jumbotron text-center"
>
  <div
    className="container"
  >
    <h1>
      Search GitHub Repos
    </h1>
    <p
      className="lead text-muted"
    >
      Search for GitHub repos using the following form
    </p>
    <form
      className="form-inline justify-content-center"
      id="search-form"
    >
      <label
        className="sr-only"
        htmlFor="inputSearch"
      >
        Search by
      </label>
      <input
        autoFocus={true}
        className="form-control"
        id="inputSearch"
        onChange={[Function]}
        placeholder="Search Repo"
        required={true}
        style={
          Object {
            "margin": 10,
          }
        }
        type="text"
        value="react"
      />
      <button
        className="btn btn-primary"
        onClick={[Function]}
        style={
          Object {
            "margin": 10,
          }
        }
        type="submit"
      >
        Search
      </button>
      <button
        className="btn btn-secondary"
        onClick={[Function]}
        type="reset"
      >
        Clear
      </button>
    </form>
  </div>
</section>
`;
